system-test/bigquery.ts

```diff
@@ -674,6 +679,17 @@ describe('BigQuery', () => {
       const [metadata] = await model.getMetadata();
       assert.strictEqual(metadata.friendlyName, friendlyName);
     });
+
+    it('should extract a model', () => {
```

Comment:
Let's make this an async function, and then use `await model.createExtractJob` to avoid the promise chaining.  As is, this test won't work correctly.